src/drivers/magnetometer/Kconfig

```diff
@@ -12,6 +12,7 @@ menu "Magnetometer"
         select DRIVERS_MAGNETOMETER_LIS2MDL
         select DRIVERS_MAGNETOMETER_LIS3MDL
         select DRIVERS_MAGNETOMETER_LSM303AGR
+        select DRIVERS_MAGNETOMETER_MEMSIC_MMC5983MA
```

Comment:
I would only include this in common if people are purchasing this module and plugging it into existing boards.